class UserNotFound(Exception):
    pass


class UnspecifiedError(Exception):
    pass